nbgrader/docs/source/user_guide/managing_assignment_files_jlab.ipynb

```diff
@@ -0,0 +1,911 @@
+{
```

Comment:
Same comment here as above about duplication in the documentation.

Reply:
The documentation has been updated in https://github.com/jupyter/nbgrader/pull/1588/commits/db5eb6a23a53dbf0b6f0f7c379b09d374297fae1